adafruit_mcp230xx/mcp23017.py

```diff
@@ -259,6 +259,8 @@ def io_control(self):
 
     @io_control.setter
     def io_control(self, val):
+        val |= 0b000001
+        val ^= 0b000001
```

Comment:
You're wanting to force bit 7 to 0 here? How about:
```python
    @io_control.setter
    def io_control(self, val):
        # force BANK bit to 0
        val &= ~0x80
        self._write_u8(_MCP23017_IOCON, val)
```
But it may be better to raise an exception since someone may think this gives them full access to the IOCON register, and for some reason they want to use BANK=1 mode (although not sure why that'd ever matter). The exception text could say something like "Only BANK=0 is supported".

Reply:
I'm not sure if an exception should even be needed, I don't think its a valid usecase since if you don't need the extra pins, just don't use them.
Just also mentioning that this was also in the original code and is technically a bugfix.

I think adding something to the documentation and forcing the `BANK` bit is enough.